Makefile

```diff
@@ -128,3 +128,47 @@ test-operator-integration:
 
 test-operator:
 	@(env bash $(PWD)/web-app/tests/scripts/operator.sh)
+
+models-gen-mac:
+	@swagger generate client -f ./swagger.yml -m ./models
+	@ls ./models | xargs -I {} gsed -i "2 a\
+// This file is part of MinIO Operator\n\
```

Comment:
Should we add a notice that these are generated files and should not be changed manually?

Reply:
it notice at the file. The command is make nothing to change for now. But it do that for real.